dev-java/openjfx/openjfx-11.0.11_p1-r1.ebuild

```diff
@@ -0,0 +1,226 @@
+# Copyright 2019-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
EAPI 6 is banned from the Gentoo repository, so please don't add any new EAPI 6 ebuilds.